### DISCUSSION TITLE
Build Fails with Docker Desktop Containerd support enabled

### DISCUSSION THREAD

**Discussion by joelhess** [werf/werf#6178] [General]
### Before proceeding

- [X] I didn't find a similar [issue](https://github.com/werf/werf/issues)

### Version

1.2.204

### How to reproduce

Mac OS 13.2.1
Macbook Pro M1
Docker Desktop 4.17.0
Docker Engine: 20.10.23

Enable the beta feature to use containerd for images: 
![image](https://user-images.githubusercontent.com/9028374/222768917-f35cbde1-48b3-42c5-9e03-d29e3901a268.png)

Run Werf build

### Result

`Error: phase build on image relay stage dockerfile handler failed: failed to build image for stage dockerfile with digest 17a670d31fca6b4e845be254fc87115036f7d5230f0d0ad3d267cb99: error building dockerfile with local-docker-server: Status: failed to read downloaded context: failed to load cache key: invalid response status 403, Code: 1`

### Expected result

Image Built successfully

### Additional information

A typical Docker build `docker build .` works fine.

**Comment by alexey-igrychev** [comment]
@joelhess Sorry for the delay, and thank you for the feedback.
The containerd image store feature is experimental, and Docker SDK does not work correctly in many cases.
We are waiting for the stabilization of it.

**Comment by shurup** [comment]
>  We are waiting for the stabilization of it.

Just a small update: as of today, not much has changed since "The containerd image store is an experimental feature of Docker Engine," according to [the documentation](https://docs.docker.com/storage/containerd/).
